### ISSUE TITLE
updating shared misc.xml

### ISSUE THREAD

**Issue by TrentonAdams** [dkandalov/limited-wip#34]
misc.xml is intended to have non-developer specific settings in it, i.e. settings that should be shared.  As such, it's inappropriate for settings from the Limited WIP plugin to be putting settings in that file.  It'd be best if an appropriate non-shared file was used.  It was my understanding that the entire .idea directory should be shared, and that only workspaces.xml was not supposed to be, so perhaps put it in there?  That's a user specific setup

https://intellij-support.jetbrains.com/hc/en-us/community/posts/360005054360-Get-misc-xml-from-VCS-but-do-not-track-it-by-default

Anyhow, it's cause my LimitedWIP settings to flap back and forth depending on what happens with my git repo.

**Comment by dkandalov** [comment]
I finally looked into this and it makes sense that keeping settings in `misc.xml` is not the best option when `.idea` is shared. However, I'm not sure if `workspace.xml` is a good alternative in case anyone wants to share the config 🤔 So maybe a separate `.idea/limited-wip.xml` file (at least it can be added to .gitignore)?

The whole concept of sharable vs non-sharable config is a bit fuzzy. For example, there is `.idea/gradle.xml` which contains config for test runner (platform or gradle) which is supposed to be shared but in practice I've seen several teams where this setting is different for each developer and using the convention above should be in `workspace.xml`.

**Comment by TrentonAdams** [comment]
Yeah, I think it's own config probably makes the most sense.  It has the added benefit that teams can choose if they want to share it or not.  Maybe they are all using WIP, and want to use the same techniques to software development.
